assets/cross-sell-carousel.css

```diff
@@ -0,0 +1,88 @@
+@media (min-width: 751), (max-width: 2000px) {
+
+  .cross-sell-product-title {
+    /* show only 2 lines of text */
+    display: block;
+    text-overflow: ellipsis;
+    word-wrap: break-word;
+    overflow: hidden;
+    max-height: 3.6em;
+    line-height: 1.8em;
+    /* end */
```

Comment:
This comment makes it look like it was copied from a example on the web, everyone has their own taste in comments, but in general, assume people reading your comments have a decent level of expierence, and try to comment only weird things, or places where you are breaking some sort of general or team specific convention, or to adhere to some sort of team convention

For instance for one of my clients we comment whenever we use z-index, since there is a risk about getting confused about z-index.

So yeah, in short, you don't need these comments.